py3status/modules/sysdata.py

```diff
@@ -92,27 +123,55 @@ def memory(self):
         # Results are in gigabytes
         return total_mem, used_mem, used_mem_p
 
-    def cpuTemp(self):
+    def cpuTemp(self, zone):
         """
         Tries to determine CPU temperature using the 'sensors' command.
         Searches for the CPU temperature by looking for a value prefixed
         by either "CPU Temp" or "Core 0" - does not look for or average
         out temperatures of all codes if more than one.
         """
 
-        sensors = subprocess.check_output(
-            'sensors',
-            shell=True,
-            universal_newlines=True,
-        )
-        m = re.search("(Core 0|CPU Temp).+\+(.+).+\(.+", sensors)
-        if m:
-            cpu_temp = m.groups()[1].strip()
+        if zone:
+            cpu_temp = float(subprocess.check_output(['sensors', zone])
+                             .split()[7].decode("utf-8")[1:-2])
+
         else:
-            cpu_temp = 'Unknown'
+            sensors = subprocess.check_output(
+                'sensors',
+                shell=True,
```

Comment:
`shell=True` is not needed here, .  It should be avoided anyhow as it has the potential to be abused when combined with user input even if that is not an issue here.

Consider using the new `py3.command_output()` helper function (and not just here).